tests/end_to_end/test_account.py

```diff
@@ -38,50 +72,57 @@ async def test_should_upgrade_outdated_account_on_transfer(
             new_account,
             other,
             class_hashes,
+            cleanup,
```

Comment:
use `@pytest.mark.usefixture("cleanup")` instead of requesting it as an (unused) argument.
you can also define the fixture with `@pytest.fixture(autouse=True)` if you want it to be used for all the tests